knowledge-base/pass-sqldatasource-columns-as-datasource-for-nested-data-items.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to Use Part of the Columns from a Parent SqlDataSource as DataSource for Nested Data Items
+description: When a parent data item is bound to a SqlDataSource you may still use part of the returned columns as DataSource for child data items
+type: how-to
+page_title: How to Pass SqlDataSource Columns as DataSource for Nested Data Items
+slug: pass-sqldatasource-columns-as-datasource-for-nested-data-items
+position: 
+tags: 
+ticketid: 1454415
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+When you need to pass data from a [SqlDataSource](../sqldatasource) bound to a data item to its nested [data item](../data-items), you need to pass all of the data. This is due to the fact that SqlDataSource can return only a single DataTable. Even if the query returns a DataSet, only the first of the tables will be used - [Selecting Data with the SqlDataSource component](../sql-data-source-selecting-data). In some cases, this may significantly deteriorate the performance.  
+The general approach for binding the DataSource of the child data item to parent data is explained in the 
+_Use DataObject as a datasource for nested data items (Table, List, Crosstab, Graph)_ section of 
+[How to use the ReportItem.DataObject property in expressions](../data-items-how-to-use-data-object) article. 
+
+## Suggested Workarounds
+MS SQL Server 2016 introduced the [JSON functions](https://docs.microsoft.com/en-us/sql/relational-databases/json/json-data-sql-server?view=sql-server-ver15) 
+that enable you to combine classic relational columns with columns that contain documents formatted as JSON text. This way you may 
```

Comment:
..."that contains documents formatted as JSON text". I suggest to use "data" instead of "documents"